astropy/modeling/functional_models.py

```diff
@@ -1469,6 +1551,53 @@ def bounding_box(self):
         return ((self.y_0 - dy, self.y_0 + dy),
                 (self.x_0 - dx, self.x_0 + dx))
 
+    def integral(self, axis='x'):
```

Comment:
Looks like consensus is to remove 2D implementation, so this won't be an issue when that is removed.